MentoringHomeWork/HW9/AirportSimulation/CheckIn.cs

```diff
@@ -0,0 +1,26 @@
+namespace AirportSimulation;
+
+public class CheckIn(Passenger passenger, Passport? passport)
+{
+    public void StartCheckIn()
+    {
+        Console.WriteLine("Check In By Airport");
+
+        var passportControl = new PassportControl();
+        if (!PassportControl.IsPassportValidationPassed(passenger, passport))
+        {
+            Console.WriteLine("\u274c The passenger did not pass the passport check.");
+            return;
```

Comment:
MINOR: return; usually separated by a line. Not going to duplicate this comment for all code.
```suggestion
            Console.WriteLine("\u274c The passenger did not pass the passport check.");
            
            return;
```

Reply:
resolved